src/containers/BooksForm.js

```diff
@@ -17,23 +21,28 @@ class BooksForm extends React.Component {
       title: '',
       category: '',
     };
-    this.updateInput = this.updateInput.bind(this);
-    this.updateSelect = this.updateSelect.bind(this);
+    this.handleChange = this.handleChange.bind(this);
     this.handleReset = this.handleReset.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
   }
 
-  updateInput = title => {
-    this.setState({ title });
-  }
-
-  updateSelect = category => {
-    this.setState({ category });
+  handleChange = e => {
+    const { name } = e.target;
+    const { value } = e.target;
+    this.setState(prevState => ({ ...prevState, [name]: value }));
   }
 
   handleReset = () => {
     this.setState({ title: '', category: '' });
   }
 
+  handleSubmit = e => {
+    e.preventDefault();
```

Comment:
- [ ] **[OPTIONAL]** The form used to create a book should have validations, the user shouldn't be able to create an empty book.
Let's create a book only if the data that the user entered are valid.